app/services/claims/fetch_eligible_misc_fee_types.rb

```diff
@@ -5,8 +5,8 @@ def initialize(claim)
     end
 
     def call
-      return nil unless claim
-      return nil if claim&.interim?
+      return unless claim
+      return [] if claim&.interim?
```

Comment:
into scope creep territory with the commit that promoted these methods and amends this. Are you confident changing this behaviour is not going to break other areas of the app. 
Not sure feature tests and other integration specs cover it so have will need manual testing to check that it does not change behaviour in both the form steps and the various summary pages for all claim types (interims and transfers in particular) that may rely on the existing eligibility logic.

Reply:
The `display_unused_materials_notice?` helper depends on the `eligible_misc_fee_types` method existing for any claim type and it must return an array. The correct response for an interim claim is an empty array as it is not eligible for any miscellaneous fee types.